src/nodes/array/pop.js

```diff
@@ -0,0 +1,48 @@
+const dayjs = require('dayjs')
+const AbstractNode = require('../abstract')
+
+class ArrayPop extends AbstractNode {
+
+  static metadata() {
+    return {
+      name: 'Pop',
+      code: 'array/pop',
+      type: 'modifier',
```

Comment:
shift, pop, push, unshift are not modifiers.
they change values of inputs, so they should be callables and has execution flow input/output